ga4gh/client.py

```diff
@@ -38,10 +36,13 @@ def __init__(self, urlPrefix, debugLevel, workarounds, key):
         else:
             logLevel = logging.DEBUG
         self._logger.setLevel(logLevel)
-        handler = logging.StreamHandler()
-        formatter = logging.Formatter('%(name)s: %(message)s')
-        handler.setFormatter(formatter)
-        self._logger.addHandler(handler)
+
+        requestsLog = logging.getLogger("requests.packages.urllib3")
```

Comment:
I was just hacking around here, as it's tricky getting the different loggers working together. This was the simplest way I could find that meant our own logs weren't getting written twice.

Totally open to ideas here if you can see a better way.